Sources/Rendering/OpenGL/ImageResliceMapper/index.js

```diff
@@ -1084,7 +1131,10 @@ function vtkOpenGLImageResliceMapper(publicAPI, model) {
         resGeomString = resGeomString.concat(`Image${image.getMTime()}`);
       }
       // Check to see if we can bypass oblique slicing related bounds computation
-      [orthoSlicing, orthoAxis] = isVectorAxisAligned(slicePlane.getNormal());
+      [orthoSlicing, orthoAxis] = isPlaneAxisAligned(
```

Comment:
Ignoring the direction matrix here was a performance optimization. Both, the plane and image are in the same space pre-direction, so, if the plane is axis aligned, no need for an expensive vtkCutter operation.

Reply:
I suppose, I will fix this right away since I already have a solution rather than kicking this down the road. Otherwise the optimization will remain as a dead code rarely executed.